website/src/main/java/com/google/step/data/OrganizationInfo.java

```diff
@@ -65,4 +79,43 @@ public Query getQueryForDuplicates() {
   public static OrganizationInfo createInstanceFrom(HttpServletRequest request) {
     return new OrganizationInfo(getOrgEntityFrom(request));
   }
+
+  public static List<OrganizationInfo> getOrganizationsFrom(CSVReader csvReader) throws IOException {
+    List<OrganizationInfo> organizations = new ArrayList<>();
+    String[] nextRecord = new String[3];
+    int index = 0; 
+    try {
+      while ((nextRecord = csvReader.readNext()) != null) { 
+        try {
+          OrganizationInfo item = new OrganizationInfo(getOrganizationEntityFrom(nextRecord, index++)); 
```

Comment:
Try not to increment index here, because if an error is caught, then index++ would still increment the index, but realistically, we would only want to increment the index if an entity is actually successfully created.

Please double check the entities in [Datastore](https://pantheon.corp.google.com/datastore/entities;kind=Organization;ns=__$DEFAULT$__/query/kind?project=mit-step-2020) as they are way too big

Reply:
Instead, just have the classic `index += 1;` after item is created without errors